requirements/base.txt

```diff
@@ -40,12 +40,15 @@ click==8.0.4
     #   apache-superset
     #   celery
     #   click-didyoumean
+    #   click-option-group
```

Comment:
@john-bodley Should we consider the CLI a development tool and move these dependencies to `requirements/development.txt`?

Reply:
@michael-s-molina I think the challenge with that is the `superset` CLI will be borked even if you ran `superset run` etc. Also as you can see there are other CLI dependencies (`click-didyoumean`) which have been included in the base requirements.